EveripediaNetwork/ep-api
# Docker

# Dockerize API
_2 dockerfiles, one runs the API, the second one the Indexer_

## How should this be tested?
Install docker, create a **.env** file for the environment variables (check **.sample.env**)  and run **docker compose up -d**

## Notes or observations

_API and Indexer are in different containers_

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/773
